src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -6,14 +6,39 @@
     ViewData["Title"] = "Apprentice Details";
 }
 
-
 <h1 class="govuk-heading-xl">@Model.ApprenticeName</h1>
 
-
-@*https://localhost:5001/10005077/apprentices/4MBBNM*@
-
 <div class="govuk-grid-row govuk-!-margin-bottom-2">
     <div class="govuk-grid-column-two-thirds">
+
+        @if (Model.Status == ApprenticeshipStatus.Stopped ||
```

Comment:
Presumably, this help block should also be subject to the feature toggle being on? Also, we could wrap the whole thing up in a single property rather than have all these ORs.

In any case, isn't this condition a long-winded way of saying, if the status is not _completed_?

Reply:
Changed to the following:
` 
@if (Model.isChangeOfEmployerEnabled && Model.Status != ApprenticeshipStatus.Completed)
        {
`